src/styles/components/_hamburger.scss

```
@@ -0,0 +1,113 @@
+.hamburger {
+  position: absolute;
+  left: rem(16px);
+  width: 100%;
+
+  @include respond-above('md') {
+    display: none;
+  }
+
+  &__container {
+    user-select: none;
+
+    > span {
+      position: relative;
+      display: block;
+      width: rem(24px);
+      height: rem(3px);
+      border-radius: rem(3px);
+      margin-bottom: rem(5px);
+
+      background: $dark;
+
+      transform-origin: rem(4px 0px);
+
+      @include transition(transform, 0.55s, ease);
+      @include transition(cubic-bezier(0.77, 0.2, 0.05, 1) opacity, 0.5s, ease);
+    }
+  }
+
+  & span:nth-child(3) {
+    width: rem(16px);
+  }
+
+  & span:nth-child(4) {
+    margin: 0;
+    transform-origin: 0% 100%;
+  }
+
+  &__input {
+    position: absolute;
+    z-index: $zindex-hamburger-input;
+
+    display: block;
+    width: rem(24px);
+    height: 100%;
+    margin: 0;
+    opacity: 0;
+
+    cursor: pointer;
+
+    -webkit-touch-callout: none;
+
+    &:checked {
+      width: rem(20px);
+      height: 100%;
+    }
+
+    &:checked ~ .hamburger__menu {
+      opacity: 1;
+    }
+
+    &:checked ~ span {
+      opacity: 1;
+      transform: rotate(45deg) translate(0, -1px);
+    }
+
+    &:checked ~ span:nth-child(3) {
+      opacity: 0;
+      transform: rotate(0deg) scale(0.2, 0.2);
+    }
+
+    &:checked ~ span:nth-child(4) {
+      transform: rotate(-45deg) translate(3.5px, -1px);
+    }
+  }
+
+  &__menu {
+    position: absolute;
+
+    display: block;
+    box-sizing: border-box;
+    box-shadow: 0 rem(2px 2px) 0 box-shadow-opacity(0.16);
+    width: 100%;
+    padding: rem(32px) rem(60px);
+    margin: 0 0 0 rem(-16px);
+    opacity: 0;
+
+    background-color: $light;
+    list-style-type: none;
+    -webkit-font-smoothing: antialiased;
+    /* to stop flickering of text in safari */
+
+    @include transition(opacity, 0.25s, ease-in-out);
+  }
+
+  &__menu-list {
+    box-sizing: border-box;
+    padding: rem(16px) 0;
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $silver;
+    }
+  }
+
+  &__menu-link {
```

Since we already have `footer__link`, do you think we should keep it like `hamburger__link` also?

```suggestion
  &__link {
```